fixtures/prolights/versapar.json

```diff
@@ -0,0 +1,200 @@
+        "color": "Blue"
+      }
+    },
+    "White": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "White"
+      }
+    },
+    "Zoom": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "Zoom",
+        "angleStart": "narrow",
+        "angleEnd": "wide"
+      }
+    },
+    "Dimmer": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Strobe": {
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "dmxRange": [0, 11],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [12, 255],
+          "type": "StrobeSpeed",
+          "speedStart": "0Hz",
+          "speedEnd": "1000Hz"
+        }
+      ]
+    },
+    "Color Macros": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 15],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [16, 255],
+          "type": "ColorPreset"
+        }
+      ]
+    },
+    "Shutter / Autospeed": {
+      "defaultValue": 255,
+      "capabilities": [
+        {
+          "dmxRange": [0, 11],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [12, 255],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Pulse",
+          "speedStart": "0Hz",
+          "speedEnd": "1000Hz",
+          "durationStart": "long",
+          "durationEnd": "short"
+        }
+      ]
+    },
+    "DIMMER SPEED": {
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "dmxRange": [0, 51],
+          "type": "Speed",
+          "speed": "fast",
+          "comment": "Present speed from display menu"
+        },
+        {
+          "dmxRange": [52, 101],
+          "type": "Speed",
+          "speed": "stop",
+          "comment": "Mode Off"
+        },
+        {
+          "dmxRange": [102, 152],
+          "type": "Speed",
+          "speed": "fast",
+          "comment": "Mode 1 fast"
+        },
+        {
+          "dmxRange": [153, 203],
+          "type": "Speed",
+          "speed": "fast",
+          "comment": "Mode 2 Middle"
+        },
+        {
+          "dmxRange": [204, 255],
+          "type": "Speed",
+          "speed": "slow",
+          "comment": "Mode 3 slow"
+        }
+      ]
+    }
+  },
+  "modes": [
+    {
+      "name": "5 CH",
+      "shortName": "5ch",
+      "channels": [
+        "Red",
+        "Green",
+        "Blue",
+        "White",
+        "Zoom"
+      ]
+    },
+    {
+      "name": "7 CH",
+      "shortName": "7ch",
+      "channels": [
+        "Dimmer",
+        "Red",
+        "Green",
+        "Blue",
+        "White",
+        "Strobe",
+        "Zoom"
+      ]
+    },
+    {
+      "name": "10 CH",
+      "shortName": "10ch",
+      "channels": [
+        "Dimmer",
+        "Red",
+        "Green",
+        "Blue",
+        "White",
+        "Color Macros",
+        "Strobe",
+        "Shutter / Autospeed",
```

Comment:
The manual has 7="STROBE / AUTO SPEED" and "AUTO SPEED SLOW TO FAST" (switched?), 8="AUTO MODE".  I think this fixture file put (half of) the former on both these channels, and forgot the latter.